mteb/models/model2vec_models.py

```diff
@@ -233,6 +337,32 @@ def encode(
     public_training_data=None,
 )
 
+potion_multilingual_128m = ModelMeta(
+    loader=partial(
+        Model2VecWrapper,
+        model_name="minishlab/potion-multilingual-128M",
+    ),
+    name="minishlab/potion-multilingual-128M",
+    languages=_POTION_MULTILINGUAL_128M_LANGUAGES,
+    open_weights=True,
+    revision="38ebd7f10f71e67fa8db898290f92b82e9cfff2a",
+    release_date="2025-05-23",
+    n_parameters=128 * 1e6,
+    memory_usage_mb=489,
+    max_tokens=np.inf,
+    embed_dim=256,
+    license="mit",
+    similarity_fn_name="cosine",
+    framework=["NumPy"],
```

Comment:
I suppose all of these are sentence transformers compatible as well. I would probably add that to the framework (even though the implementation in mteb uses model2vec)